addons/mail/views/assets.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <template id="_assets_primary_variables" inherit_id="web._assets_primary_variables">
+        <template id="_assets_helpers" inherit_id="web._assets_helpers">
```

Comment:
Any reason you renamed on this line?

Reply:
Yes, including our variables file in primary variables caused a crash because bootstrap variables are not yet loaded

```xml
    <template id="_assets_primary_variables">
        <link rel="stylesheet" type="text/scss" href="/web/static/src/scss/primary_variables.scss"/>
    </template>

    <template id="_assets_secondary_variables">
        <link rel="stylesheet" type="text/scss" href="/web/static/src/scss/secondary_variables.scss"/>
    </template>

    <template id="_assets_helpers">
        <t t-call="web._assets_utils"/>
        <t t-call="web._assets_primary_variables"/>
        <t t-call="web._assets_secondary_variables"/>

        <t t-raw="0"/>

        <link rel="stylesheet" type="text/scss" href="/web/static/lib/bootstrap/scss/_variables.scss"/>
    </template>
```

Reply:
Ok, good enough reason. Looks indeed the correct way to do this then.